.github/workflows/codeql-analysis.yml

```diff
@@ -12,7 +12,7 @@ permissions: read-all
 
 concurrency:
   group: build-${{ github.event.pull_request.number || github.ref }}-${{github.workflow}}
-  cancel-in-progress: true
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
```

Comment:
I remembered that you changed it to ``${{ github.ref != 'refs/heads/main'}}`` is this a regression while you're changing group stanza ?

Reply:
Thanks! Yes, was an accident in moving things around, fixed now